feature-libs/pickup-in-store/core/connectors/pickup-location.adapter.ts

```diff
@@ -16,4 +20,30 @@ export abstract class PickupLocationAdapter {
    * @param storeName The store name to get details for
    */
   abstract getStoreDetails(storeName: string): Observable<PointOfService>;
+  /**
+   * Set Pickup Option to Delivery
+   * @param cartId
+   * @param entryNumber
+   * @param userId
+   * @param payload
+   */
+  abstract setPickupOptionDelivery(
+    cartId: string,
+    entryNumber: number,
+    userId: string,
+    payload: SetPickupOptionDeliveryPayload
+  ): Observable<any>;
+  /**
+   * Set Pickup Option to Pickup in Store
+   * @param cartId
+   * @param entryNumber
+   * @param userId
+   * @param payload
+   */
+  abstract setPickupOptionInStore(
+    cartId: string,
+    entryNumber: number,
+    userId: string,
+    payload: SetPickupOptionInStorePayload
+  ): Observable<any>;
```

Comment:
We can improve the js docs for these. It should be made clear that this is for amending products already in the cart.